docs/copy.md

```diff
@@ -1,4 +1,4 @@
-# copy(src, dest, [options, callback])
+# copy(src, dest[, options, callback])
```

Comment:
If we follow the Node pattern we can also add optional options:

```
# copy(src, dest[, options][, callback])

Reply:
@JPeer264 that's a good point! I'll update it then.